metricbeat/values.yaml

```diff
@@ -337,4 +337,11 @@ deployment:
   - name: cert
     mountPath: "/etc/pki/tls/certs/SectigoRSADomainValidationSecureServerCA.crt"
     readOnly: true
-    subPath: SectigoRSADomainValidationSecureServerCA.crt
\ No newline at end of file
+    subPath: SectigoRSADomainValidationSecureServerCA.crt
+
+secrets:
+  logzioShippingToken: ""
+  logzioRegion: " "
+  clusterName: ""
+  kube-state-metrics-port: "8080"
```

Comment:
kube-state-metrics port and values shouldn't actually be required to be stored in secret, should they? Surely service discovery is sufficient for this.